modules/va_facilities/README.yml

```diff
@@ -66,6 +66,22 @@ info:
 tags:
   - name: facilities
     description: VA Facilities API
+servers:
+- url: https://dev-api.va.gov/services/va_facilities/{version}
+  description: VA.gov API development environment
+  variables:
+    version:
+      default: v0
+- url: https://staging-api.va.gov/services/va_facilities/{version}
+  description: VA.gov API staging environment
+  variables:
+    version:
+      default: v0
```

Comment:
Let's take staging out of this. We don't really want people using that environment anymore.